src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
       });
   }
 
+  handleSelectChange = selectedOption => {
+    this.alertsSelectionArray = selectedOption;
+    this.setState({
+      alertSignUps: this.alertsSelectionArray
+        .map(item => item.full_fips_code)
+        .join(','),
+    });
+  };
+
   render() {
     const { stateId, county } = this.props;
+
+    const stateDataset = US_STATE_DATASET.state_dataset.map(state => {
+      return {
+        ...state,
+        full_fips_code: state.state_fips_code + '000',
+      };
+    });
+    const countyDataset = [];
+
+    each(US_STATE_DATASET.state_county_map_dataset, (value, key) => {
+      countyDataset.push(...value.county_dataset);
+    });
+
+    const autocomplete_options = stateDataset.concat(countyDataset);
+
     return (
       <StyledNewsletter>
         <form
           ref={f => (this.form = f)}
           className="js-cm-form"
           id="subForm"
           method="post"
-          data-id="2BE4EF332AA2E32596E38B640E90561930C9C3A433D015D9D4BD88E99175E51395EF5EBFFD527179E032AC15455BB1208D87A6CE87843E524B0EA520CBFF446E"
+          data-id="2BE4EF332AA2E32596E38B640E905619E90CD5DAC48A878CDEBFFE3B420D8CD24E4AEABAB52A4CE3526219C7A966AE965B84F99C823C89EFF1F01B28DE4F975E"
```

Comment:
FWIW- Since you had to puzzle out how these IDs and hidden form element things work, it might be worth adding some comments somewhere in this file to help the next unfortunate soul who has to mess with this stuff. 😛

Reply:
done added some comments above